exchanges/gateio/gateio_websocket_request_futures.go

```diff
@@ -101,7 +101,7 @@ func (g *Gateio) WebsocketFuturesCancelAllOpenFuturesOrders(ctx context.Context,
 		return nil, fmt.Errorf("%w: %s", order.ErrSideIsInvalid, side)
 	}
 
-	params := struct {
+	params := &struct {
 		Contract currency.Pair `json:"contract"`
 		Side     string        `json:"side,omitempty"`
 	}{Contract: contract, Side: side}
```

Comment:
🥃  It's kinda weird to have this `wsCancelAllFuturesOrders` type just anonymously hidden here, and also in other examples around here.

Just saying.
![kermit](https://us.tonies.com/cdn/shop/files/Tonies-PDP-Assets-kermit-single.jpg?v=1734730750&width=3000)